packages/package_info_plus/package_info_plus_web/lib/package_info_plus_web.dart

```diff
@@ -58,3 +58,16 @@ class PackageInfoPlugin extends PackageInfoPlatform {
     }
   }
 }
+
+extension _UriOrigin on Uri {
+  /// Get origin.
+  ///
+  /// This is different from [Uri.origin] because that has checks to prevent
+  /// non- http/https use cases.
+  String get _origin {
+    if (isScheme('chrome-extension')) {
+      return '$scheme://$host:$port';
```

Comment:
Don't we need this logic here?
```
int? port = _port;
if (port == null) return "$scheme://$host";
return "$scheme://$host:$port";
```

Reply:
a few observations:
1. i don't think chrome extensions support ports, although i wasn't able to find anything backing this. (one theory is that it's not a server, so what will the port manager be? unlike nginx used in backend)
2. actually, having and not having the port seems to be same in case of chrome extensions. if you pass a port in the url, chrome just removes it once you press enter.

so given this, i think it's safe to remove the port from here too. (i didn't do this before cause i wasn't completely sure of the above two observations).